docs/DeveloperGuide.md

```diff
@@ -841,7 +843,28 @@ Precondition: CS2100 TA opens ProgrammerError for the first time
 
     Use case resumes at step 3.
 
-### <a name="Use Case3"></a> **Use case: UC3 View a student record**
+### <a name="Use Case3"></a> **Use case 3: UC3 Upload student records**
```

Comment:
I think there is inconsistent use of "Use case" and "Use case <Number>".
Might want to consider standardising.

Reply:
Propose for this line to be
`### <a name="Use Case3"></a> **Use case: UC3 Upload student records**`

Reply:
Should be fixed now!